test/tests/functional/pbs_qsub_remove_files.py

```diff
@@ -36,6 +36,7 @@
 # trademark licensing policies.
 
 from tests.functional import *
+from ptl.lib.pbs_ifl_mock import ATTR_R
```

Comment:
I don't understand this specific inclusion. You are using ATTR_J and others which seem to work fine.

Reply:
Fixed in new commit rc2 https://github.com/PBSPro/pbspro/pull/892/commits/90b8b4df18442ed3f8016032e9c1cc43f83cdf5f